__DEFINES/planes+layers.dm

```diff
@@ -144,7 +144,7 @@ What is the naming convention for planes or layers?
 
 #define HUMAN_PLANE 			9			// For Humans that are standing up.
 
-//#define THIS_SPACE_FOR_RENT!	10
+#define VAMP_ANTAG_HUD_PLANE	10
```

Comment:
This shouldn't be necessary, as the antag hud @DeityLink put in place just flicks through each icon available anyway.

Reply:
I need it for the client image of the blood count.